helm/cluster-gcp/templates/_control_plane.tpl

```diff
@@ -118,7 +118,7 @@ metadata:
 spec:
   template:
     spec:
-      image: {{ .Values.gcp.baseImage }}
+      image: {{ include "vmImage" $ }}
```

Comment:
just for me to understand a bit better, what is the difference beteween $ and $global in machine_deployments.tpl

Reply:
See here: https://github.com/giantswarm/cluster-gcp/blob/0965adb7152b116a2b8842ee5d275fd5c1271cb3/helm/cluster-gcp/templates/_machine_deployments.tpl#L2-L3

The scope is changed and `$global` is used elsewhere in the block so I was keeping it consistent. I'm not sure it's needed over just `$` though.

Reply:
Cool thanks 👍🏻, that was the missing thing I haven't seen.